addon/mixins/comp.js

```diff
@@ -0,0 +1,25 @@
+import Ember from 'ember';
+
+const {
+  get: get,
+  set: set,
+  Mixin,
+  on
+} = Ember;
+
+export default Mixin.create({
+  deprecations: on('didInsertElement', function() {
```

Comment:
I don't think this needs to be a mixin, it can be moved into the lazy image component itself